ChatApp/ChatApp.xcodeproj/project.pbxproj

```diff
@@ -2119,7 +2123,7 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_ENTITLEMENTS = ChatApp/SupportingFiles/ChatApp.entitlements;
-				CODE_SIGN_IDENTITY = "iPhone Developer";
+				CODE_SIGN_IDENTITY = "Apple Development: Jan Pacek (XT97TS9CAT)";
```

Comment:
Is this on purpose? 👀

Reply:
@danpecher Good catch 💪 But it is on purpose. That is our STRV development certificate.